fuse/join_events_personnel_tables.py

```diff
@@ -0,0 +1,20 @@
+import deba
+import pandas as pd
+
+
+def join_event_and_personnel():
+    dfa = pd.read_csv(deba.data("fuse/event.csv"))
+    dfb = pd.read_csv(deba.data("fuse/personnel.csv"))
+
+    df = pd.merge(
+        dfa[["agency", "uid"]],
+        dfb[["first_name", "last_name", "middle_name", "uid"]],
+        on="uid",
+        how="outer",
+    )
+    return df.dropna().drop_duplicates(subset=["uid"])
+
+
+if __name__ == "__main__":
+    df = join_event_and_personnel()
+    df.to_csv(deba.data("fuse/events_and_personnel_merged.csv"), index=False)
```

Comment:
This file seems redundant.

Reply:
Should this function be moved to the match script where I match post and events/personnel?

Reply:
I said it is redundant because I don't see the output "fuse/events_and_personnel_merged.csv" used anywhere.

Reply:
Thanks.